nvtabular/io/hugectr.py

```diff
@@ -23,7 +23,10 @@
 class HugeCTRWriter(ThreadedWriter):
     def __init__(self, out_dir, **kwargs):
         super().__init__(out_dir, **kwargs)
-        self.data_paths = [os.path.join(out_dir, f"{i}.data") for i in range(self.num_out_files)]
+        if self.use_guid:
+            self.data_paths = [os.path.join(out_dir, f"{i}i.{guid()}.data") for i in range(self.num_out_files)]
+        else:
+            self.data_paths = [os.path.join(out_dir, f"{i}.data") for i in range(self.num_out_files)]
```

Comment:
Looks like black doesn't like this : 
```suggestion
        if self.use_guid:
            self.data_paths = [
                os.path.join(out_dir, f"{i}i.{guid()}.data") for i in range(self.num_out_files)
            ]
        else:
            self.data_paths = [
                os.path.join(out_dir, f"{i}.data") for i in range(self.num_out_files)
            ]
```

also you might want to install the precommit hook here to catch this automatically:

```
#  from https://pre-commit.com/ - inside the nvtabular go:
pip install precommit
precommit install
```